### PR TITLE
Fixed windows CR LF filter

### PR DESCRIPTION
**Pull request by SilentSys**

### CODE REVIEW

File: connections/sim/stream_logger.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def _read_and_log(self, stream_gen):
     def _crcrlfFilter(self, stream):
         while True:
             c = stream.read(1)
-            if c == CR and stream.peek(1)[0] == LF:
+            if c[0] == CR and stream.peek(1)[0] == LF:
```
Comment (SilentSys) [inline connections/sim/stream_logger.py:45]: This is actually the entire fix. The rest is just misc stuff.

Comment (jesseli2002) [reply connections/sim/stream_logger.py:45]: Fix aside, I'm curious how this ever worked, if at all

**Review by jesseli2002 — APPROVED** [review]

File: connections/sim/stream_logger.py (hunk 1 above)
Comment (SilentSys) [reply connections/sim/stream_logger.py:45]: Not sure, but it will never happen again (hopefully (see new commit))

**Review by DSep — APPROVED** [review]
Sweet; looking forward to trying this today on our call and reporting what the state of SIM is

**Pull request merged**